app/_hub/kong-inc/vault-auth/index.md

```diff
@@ -108,25 +108,27 @@ A Vault object represents the connection between Kong and a Vault server. It def
 Vault objects can be created via the following HTTP request:
 
 ```bash
-$ curl -X POST http://kong:8001/vaults \
-  --data name=kong-auth \
-  --data mount=kong-auth \
-  --data protocol=http \
-  --data host=127.0.0.1 \
-  --data port=8200
-  --data token=s.m3w9gdV0uMDYFpMgEWSB2mtM
+$ curl --request POST \
+  --url http://localhost:8001/vaults \
+  --header 'Content-Type: multipart/form-data' \
+  --form name=kong-auth \
+  --form mount=kong-auth \
+  --form protocol=http \
+  --form host=127.0.0.1 \
+  --form port=8200 \
+  --form vault_token=<token>
 HTTP/1.1 201 Created
```

Comment:
I'd suggest splitting out the request and the result into two codeblocks. That way, someone can click the copy icon to copy the request only.

Reply:
Done in commit 20002d3